Initial commit

MartynoSiela opened the pull request:
Create common.js, login.js, timeLogging.js to store PO's. Refactor sourcery.js to include newly created PO's. Check for user name to verify that logged in successfully.

MartynoSiela committed:
Initial commit

Create common.js, login.js, timeLogging.js to store PO's. Refactor sourcery.js to include newly created PO's. Check for user name to verify that logged in successfully.

linavegiene reviewed (commented):
Looks OK. I will not approve as I don't wan't you to merge this branch to develop. 

You can go on to create a new branch for homework task.

p.s. I noticed you got rid of code to verify menu, why?

MartynoSiela committed:
Make TC Admin user logs in

Create PO's menuBar, activeMenuField in common.js.
Create variables menuButtons, colorBlue in sourcery.js.
Add code to check if all elements of menuButtons array are present.
Add code to check if activeMenuField is 'Time Logging' and color is blue.

MartynoSiela committed:
Format code in Visual Studio to be consistent.

MartynoSiela committed:
TC-MS Admin creates new task

Add code to create task with name, description, rate provided as variables. Add code to go to all tasks, filter and check if task was created (check the title).
